feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ConversationAutoCompletionE2eTest.kt

```diff
@@ -0,0 +1,57 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.conversation
+
+import de.tum.informatics.www1.artemis.native_app.core.common.test.EndToEndTest
+import de.tum.informatics.www1.artemis.native_app.core.common.test.DefaultTestTimeoutMillis
+import de.tum.informatics.www1.artemis.native_app.core.test.test_setup.DefaultTimeoutMillis
+import de.tum.informatics.www1.artemis.native_app.core.common.test.testServerUrl
+import de.tum.informatics.www1.artemis.native_app.core.model.account.User
+import de.tum.informatics.www1.artemis.native_app.feature.login.test.user1Username
+import de.tum.informatics.www1.artemis.native_app.feature.login.test.user2Username
+import de.tum.informatics.www1.artemis.native_app.feature.login.test.user3Username
+import de.tum.informatics.www1.artemis.native_app.feature.metistest.ConversationBaseTest
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.util.Logger
+import kotlin.test.assertTrue
+import kotlin.test.assertEquals
+import kotlin.time.Duration.Companion.milliseconds
+
+
+@Category(EndToEndTest::class)
+@RunWith(RobolectricTestRunner::class)
+class ConversationAutoCompletionE2eTest : ConversationBaseTest() {
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `test GIVEN users are registered in a course WHEN requesting auto complete users THEN the registered users are returned`() {
```

Comment:
I know the end to end tests right now do not run in the CI, but were you able to verify that this test runs locally?

Reply:
That's actually the problem, the error in the CI and locally were the same, so the tests also do not run locally.  #66 should take care of this and I'll take a deeper look into it as soon as possible next week.

Reply:
The tests should run locally as long as you have an Artemis instance running listening on port 80. You do not need to execute them inside docker.

Reply:
Good to know, I'll try that 👍

Reply:
Sorry for the delay. I tested it with the new docker containers and the test passed